### DISCUSSION TITLE
Native Speed for Recognize App (Tensorflow)

### DISCUSSION THREAD

**Discussion by frederikb96** [nextcloud/all-in-one#1616] [Ideas] [answered]
The Recognize App is already supported with [sub-native speed](https://github.com/nextcloud/recognize#requirements) (#616).

But I was unable to run the native mode, since TensorFlow got not installed due to the Alpine image and unsupported glibc. My processor supports the AVX instruction set, so it would be very nice if there is a way to enable the native mode.

I tried to [manually install ](https://github.com/nextcloud/recognize/wiki/Manual-install) TensorFlow within the Alpine container by using some Alpine glibc alternatives, but I did not succeed.

One possibility would simply be to use the Ubuntu container, but I guess that this is not at all desired, so will probably not happen.

**Comment by szaimen** [comment, marked as answer]
> One possibility would simply be to use the Ubuntu container, but I guess that this is not at all desired, so will probably not happen.

Yes, we will stay with the Alpine container due to size and security concerns. 

One way to go forward with this would be asking for this feature upstream in the `gcompat` repo...
